portal-test/src/com/liferay/portal/test/rule/ExpectedLogs.java

```diff
@@ -23,6 +23,6 @@
 
 	public String level();
 
-	public Class<?> loggerClass();
```

Comment:
@rafaprax can you still keep this ```public Class<?> loggerClass();``` ? it is still preferred when the Class is accessible from the compiling context for type safety purpose.